docs/troubleshooting/index.rst

```diff
@@ -23,6 +23,64 @@ Looking at logs
 
 See: `The Littlest JupyterHub documentation <https://the-littlest-jupyterhub.readthedocs.io/en/latest/troubleshooting/logs.html>`_.
 
+The main components to have a look at are:
+
+- the JupyterHub instance
+- Traefik (proxy)
+- the user server
+
+The logs for the hub and proxy can be inspected with:
+
+.. code-block:: bash
+
+   sudo journalctl -u jupyterhub
+   sudo journalctl -u traefik
+
+
+.. note::
+
+   To follow the logs of the services and print more content on the screen, you can combine the ``-f`` and ``-n`` options:
```

Comment:
Could remind the user what -f and -n options are for?
For instance:
```
To follow the logs of the services and print more content on the screen, you can combine the ``-f`` (to print the end of the logs) and ``-n`` (to specify the number of lines to display) options:
```

Reply:
Sounds good, done in https://github.com/plasmabio/plasma/pull/200/commits/c62cca105662f613b005a311fa2914ffb9b306a4